featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -278,7 +284,10 @@ def filt(f):
 
         new_features.sort(key=lambda f: f.get_depth())
 
+        before_len = len(new_features)
         new_features = self._filter_features(new_features)
+        if verbose and before_len > len(new_features):
```

Comment:
what does this look like when it gets written? 

I like the main change, but not sure it's that helpful to have this print out as things are removed.

Reply:
It actually looks pretty good I think, try running it and check it out.

Basically the progress bar is like:

[45 features] 100 features/s

Eventually the feature building stops and the progress bar stops, then above the progress bar the number removed is written:

Removed 2 features with undesired variable types
Removed 2 features from drop_contains list
[45 features] 100 features/s

Reply:
can you screenshot?